modules/RTC/JitsiRemoteTrack.js

```diff
@@ -51,7 +85,7 @@ JitsiRemoteTrack.prototype.setMute = function (value) {
         this.stream.muted = value;
 
     this.muted = value;
-    this.eventEmitter.emit(JitsiTrackEvents.TRACK_MUTE_CHANGED);
+    this.eventEmitter.emit(JitsiTrackEvents.TRACK_MUTE_CHANGED, this);
```

Comment:
Because it makes implementation of such listeners harder and single listener instance can not be reused. I basically have to create separate listener instance for every track and store it somewhere in order to unregister it later. Like here for example I have one instance, but would have to create some map or whatever to store all of these:
https://github.com/jitsi/lib-jitsi-meet/blob/master/modules/connectivity/ParticipantConnectionStatus.js#L236

Without passing the source object event listener interface feels incomplete.